packages/web/src/scss/components/UNSTABLE_Toggle/_theme.scss

```
@@ -1,21 +1,36 @@
-$input-mark-disabled: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="#{$input-mark-width}" height="#{$input-mark-height}" viewBox="0 0 52 44" fill="none"><circle xmlns="http://www.w3.org/2000/svg" cx="26" cy="18" r="10" fill="#{tokens.$action-unselected-disabled}"/></svg>');
-$input-mark-disabled-checked: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="#{$input-mark-width}" height="#{$input-mark-height}" viewBox="0 0 52 44" fill="none"><circle xmlns="http://www.w3.org/2000/svg" cx="26" cy="18" r="10" fill="#{tokens.$action-selected-disabled}"/></svg>');
+$input-border-width: global-tokens.$border-width-100;
+$input-border-color: var(--#{globals.$prefix}color-action-toggle-unselected-border);
+$input-border-color-checked: var(--#{globals.$prefix}color-action-toggle-selected-border);
+$input-border-color-disabled: var(--#{globals.$prefix}color-disabled-border);
+$input-background-color: var(--#{globals.$prefix}color-action-toggle-unselected-default);
+$input-background-color-hover: var(--#{globals.$prefix}color-action-toggle-unselected-hover);
+$input-background-color-checked: var(--#{globals.$prefix}color-action-toggle-selected-default);
+$input-background-color-checked-hover: var(--#{globals.$prefix}color-action-toggle-selected-hover);
+$input-background-color-disabled: var(--#{globals.$prefix}color-disabled-background);
+$input-border-radius: global-tokens.$radius-400;
+$input-focus-shadow: global-tokens.$focus;
+
+$input-mark-size: 20px;
+$input-mark-position-y: global-tokens.$space-100;
+$input-mark-position-x: global-tokens.$space-100;
+$input-mark-border-radius: 50%;
+$input-mark: var(--#{globals.$prefix}toggle-input-mark-background-image-url);
```

Maybe clarify the type of the value?

```suggestion
$input-mark-image: var(--#{globals.$prefix}toggle-input-mark-background-image-url);
```